pkg/lib/cockpit-components-plot.jsx

```diff
@@ -404,8 +404,10 @@ export const SvgPlot = ({ title, config, plot_state, plot_id, className }) => {
             if (x < m_left) x = m_left;
             if (x > w - m_right) x = w - m_right;
             setSelection({
-                start: selection.start, stop: x,
-                left: Math.min(selection.start, x), right: Math.max(selection.start, x)
+                start: selection.start,
+                stop: x,
+                left: Math.min(selection.start, x),
+                right: Math.max(selection.start, x)
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17890-20221108-090107-5c6c50b0-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)